examples/case_studies/conditional_autoregressive_priors.myst.md

```diff
@@ -0,0 +1,350 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python [conda env:spatial_pymc_env]
+  language: python
+  name: conda-env-spatial_pymc_env-py
+substitutions:
+  extra_dependencies: bambi seaborn
```

Comment:
yes, only the metadata and these two lines are enough. You can see in the preview how this two lines add into the rendered notebook a predefined template with install instructions with pip, conda and from within jupyter: https://pymcio--547.org.readthedocs.build/projects/examples/en/547/case_studies/conditional_autoregressive_priors.html

Reply:
i'm not clear on where I can add the metadata. The substitutions lines only appears in the myst file (as far as I can tell) and I thought that's entirely generated from the .ipynb. 

Also, if it matters we don't need seaborn or bambi for this one. Just geopandas and pylibsal. Both should be installed with conda-forge.

Reply:
The section I linked of the style guide covers how to edit the metadata from jupyter. This (together with cell level metadata which we also use) is one of the main reasons we use myst files in addition to ipynb. The ipynb is needed to store outputs and render the website properly, myst ones are needed for proper reviews to be possible (this might change with the improvements in jupyer diffs github is doing but for now I think they are still needed)

Reply:
I'm still pretty unsure - the .ipynb metadata now reads:

```
  "substitutions": {
    "extra_dependencies": "geopandas libpysal"
```

but the document preview is unchanged. What might I be missing?

Reply:
It has been updated, you might have been a bit too eager and not waited for the rtd check (has status mark below) finished. I do see the new libraries now.

This behaviour is the one that appears in the style guide (and still works) but was recently deprecated. It should be:

```
  "myst": {
    "substitutions": {
      "extra_dependencies": "geopandas libpysal"
    }
  },
```